src/Symfony/Component/Form/Resources/translations/validators.az.xlf

```diff
@@ -72,7 +72,7 @@
             </trans-unit>
             <trans-unit id="113">
                 <source>Please select a valid locale.</source>
-                <target>Zəhmət olmasa doğru bir dil seçin.</target>
+                <target>Zəhmət olmasa doğru bir yer seçin.</target>
```

Comment:
This says "place"? That is also not 100% correct. 

What about "Zəhmət olmasa düzgün yeri seçin."?

Reply:
@Nyholm Yes, unfortunately there is no exact translation for the term yet. It is mostly translated as "dil" (https://azerdict.com/english/locale) which means _language_.

Reply:
Okey. Then Im happy with this. 

Thank you for checking